doc/dev/testing/travis-build.md

```diff
@@ -86,12 +86,18 @@ The Go, Ansible, and Helm tests then differ in what tests they run.
 ### Helm Tests
 
 1. Run [helm e2e tests][helm-e2e].
-    1. Build base helm operator image from [`test/helm-operator`][helm-base].
-    2. Create and configure a new helm type nginx-operator.
-    3. Create cluster resources.
-    4. Wait for operator to be ready.
-    5. Create nginx CR and wait for it to be ready.
-    6. Delete nginx CR and verify that finalizer (which writes a message in the operator logs) ran.
+    1. Create base helm operator image by running [`hack/image/helm/scaffold-helm-image.go`][helm-base].
```

Comment:
```suggestion
    1. Create base helm operator project by running [`hack/image/helm/scaffold-helm-image.go`][helm-base].
```

Reply:
I fixed this and made the same change in the ansible-operator tests.